recipe/bld.bat

```diff
@@ -0,0 +1,115 @@
+:: @echo off
+:: Try using bootstrapped ZIG to build ZIG for Conda environment
+set "MSVC_TARGET=x86_64-windows-msvc"
+set "GNU_TARGET=x86_64-windows-gnu"
+set "MCPU=native"
+
+:: set "CC=%BUILD_PREFIX%\Library\ucrt64\bin\gcc"
+:: set "CXX=%BUILD_PREFIX%\Library\ucrt64\bin\g++"
+
+:: Configure CMake in build directory
+set "SOURCE_DIR=%SRC_DIR%\zig-source"
+set "CONFIG_DIR=%SRC_DIR%\_config"
+
+call :configZigCmakeBuildCLANG "%CONFIG_DIR%" "%SRC_DIR%\_conda-cmake-built"
+if %ERRORLEVEL% neq 0 exit /b %ERRORLEVEL%
+
+:: call :bootstrapZigWithZIG "%SRC_DIR%\_conda-bootstrap" "%SRC_DIR%\zig-binary\zig.exe" "%SRC_DIR%\_conda-bootstrapped"
+:: if %ERRORLEVEL% neq 0 (
+::   echo "Failed to bootstrap ZIG"
+::   exit /b %ERRORLEVEL%
+:: )
+
+call :buildZigWithZIG "%SRC_DIR%\_conda-zig-build" "%SRC_DIR%\_conda-cmake-built\bin\zig.exe" "%SRC_DIR%\_conda-final"
+if %ERRORLEVEL% neq 0 (
+    echo "Failed to build ZIG"
+    exit /b %ERRORLEVEL%
+    )
+
+echo Copying ZIG to %PREFIX%
+mkdir %PREFIX%\bin
+mkdir %PREFIX%\lib
+mkdir %PREFIX%\doc
+copy %SRC_DIR%\_conda-final\zig.exe %PREFIX%\bin\zig.exe
+xcopy /E %SRC_DIR%\_conda-final\lib %PREFIX%\lib\ > nul
+xcopy /E %SRC_DIR%\_conda-final\doc %PREFIX%\doc\ > nul
+
+dir %PREFIX%\lib
+
+:: Exit main script
+GOTO :EOF
+
+:: --- Functions ---
+
+:configZigCmakeBuildCLANG
+setlocal
+set "_build_dir=%~1"
+set "_zig_install_dir=%~2"
+
+mkdir %_build_dir%
+if %ERRORLEVEL% neq 0 exit /b %ERRORLEVEL%
+cd %_build_dir%
+  set "_prefix=%PREFIX:\=\\%"
+  set "_zig_install_dir=%_zig_install_dir:\=\\%"
+
+  set "CLANG_MAXIMUM_CONCURRENT_JOBS=1"
```

Comment:
Maybe we should limit [CMake parallelism]( https://cmake.org/cmake/help/latest/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html ) as well

```suggestion
  set "CLANG_MAXIMUM_CONCURRENT_JOBS=1"
  set "CMAKE_BUILD_PARALLEL_LEVEL =1"
```